saleor/graphql/order/bulk_mutations/draft_orders.py

```diff
@@ -34,14 +34,14 @@ def clean_instance(cls, info, instance):
             )
 
 
-class DraftOrderLinesBulkDelete(ModelBulkDeleteMutation):
+class DeprecatedDraftOrderLinesBulkDelete(ModelBulkDeleteMutation):
     class Arguments:
         ids = graphene.List(
             graphene.ID, required=True, description="List of order lines IDs to delete."
         )
 
     class Meta:
-        description = "Deletes order lines."
+        description = "DEPRECATED: Will be removed in Saleor 4.0."
```

Comment:
The description of the mutation should be left.
```suggestion
        description = "Deletes order lines. DEPRECATED: Will be removed in Saleor 4.0."
```